python/ct/proto/client.proto

```diff
@@ -205,3 +205,9 @@ message SignedCertificateTimestamp {
                                   (tls_opts).max_length = 0xffff ];
   optional DigitallySigned signature = 5;
 }
+
+message SignedCertificateTimestampList {
+  repeated bytes sct_list = 1 [ (tls_opts).min_length = 1,
```

Comment:
As you point out, I think it's not necessary - I just want to make sure a few SCTLists are encoded similarly by other encoders.